app/controllers/lti_nrps_controller.rb

```diff
@@ -64,11 +64,13 @@ def request_access_token
   end
 
   # NRPS endpoint for Autolab to send an NRPS request to LTI Advantage Platform
-  action_auth_level :send_nrps_request, :instructor
-  def send_nrps_request
+  action_auth_level :sync_roster, :instructor
+  def sync_roster
+    params.require(:lcd_id)
```

Comment:
This seems to be undoing your previous change, should be reverted (remove `params.require(:lcd_id)`)

Reply:
Yeah, removed.